packages/magento-wishlist/queries/GuestWishlist.graphql

```diff
@@ -1,12 +1,27 @@
 query GuestWishlist {
-  guestWishlist @client {
-    __typename
-    items {
-      __typename
-      sku
-      url_key
-      selected_options
-      quantity
+  customer @client {
```

Comment:
`customer @client` doesn't make sense, why would we use a `@client` directive here?